src/dynamics/video_player/player/player.html

```diff
@@ -35,16 +35,6 @@
             ba-noengagenext="{{noengagenext}}"
             ba-gallerysidebar="{{gallerysidebar}}"
         ></ba-{{dynnext}}>
-        <div class="{{css}}-tooltips-list-container"
-             ba-repeat="{{tooltip :: tooltips}}"
-        >
-            <ba-{{dyntooltip}}
-                ba-if="{{tooltip.tooltiptext}}"
-                ba-tooltip="{{tooltip}}"
-                ba-playing="{{playing}}"
-                ba-adsplaying="{{adsplaying}}"
-            ></ba-{{dyntooltip}}>
-        </div>
```

Comment:
@Josewd removing it from here will hide the tooltip on other themes besides Kargo's. @sambua do you know if we're using tooltips on any other theme?
I was thinking we could keep this as is and not use the `tooltips` attribute for showing the "click to unmute" tooltip.

Reply:
If we don't worry about supporting tooltips on the other themes we can disregard this.

Reply:
Hi @ntshcalleia , I added an id to tooltip element on betajs, so its can easly be found on kargo video and hide it with css. the id is dynamically, so we can hide any tooltip, in this case `unmute-tooltip`

Reply:
Hi @ntshcalleia as it's in the root of the player.html so it can be used with any other themes as well. Themes manage controlbar and message components only.